docker-startup.sh

```diff
@@ -6,4 +6,11 @@ if [[ ! -d "node_modules" ]]; then
   npm install
 fi
 
-npm start
\ No newline at end of file
+#start trace allocator
+echo "${NODE_ENV:-production}"
+if [[ "${NODE_ENV:-production}" == "development" ]]; then
+  npm run dev
+  #perf record -e cycles:u -g -- npm run dev > perf.out
+else
+  npm start
+fi
```

Comment:
It is a bit weird having this. Image should be launched in production mode. Image CMD should be start and use that. If required to launch in dev mode, docker-compose.yml should have a command: for the service that use `run dev`

Reply:
removed here